src/management/src/management_frontend.hpp

```diff
@@ -259,6 +317,67 @@ class RequestHandler {
   }
 
   /**
+   * Creates an endpoint that listens for requests to retrieve info about
+   * registered Clipper applications.
+   *
+   * JSON format:
+   * {
+   *  "verbose" := bool
+   * }
+   *
+   * \return Returns a JSON string that encodes a list with info about
```

Comment:
Are you sure about this? The other methods in management_frontend.hpp have their desired json input schema shown in the same way that it is here.